5.5/root/usr/share/container-scripts/mysql/lib.sh

```diff
@@ -0,0 +1,3 @@
+function info-log {
```

Comment:
The reason why I wanted it outside of common.sh is that common.sh isn't really common across images. I wanted to have a separate file that could be directly copyable so it would be easy to keep up-to-date.